### PR TITLE
Add basic docs on secondaries

### PR DESCRIPTION
**Pull request by cajun-rat**
Change-Id: I32a304ad568898e2c21b688e67e58e869c7fc761

### CODE REVIEW

File: docs/linux-secondaries.adoc [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+Start the secondary image first with:
+
+     ../meta-updater/scripts/run-qemu-ota --no-gui --machine qemux86-64 --secondary-network secondary-image
+
+The `--secondary-network` option enables a second virtual network interface.
+
+Then start the primary:
+
+     ../meta-updater/scripts/run-qemu-ota --no-gui --machine qemux86-64 --secondary-network primary-image
+
```
Comment (lbonn) [inline docs/linux-secondaries.adoc]: note: I'm currently trying to confirm that this is indeed a systemd bug

**Comment by patrickvacek** [comment]
It's probably worth saying something about how to make sure it's actually working.

**Comment by patrickvacek** [comment]
Might also be worth specifying that the secondary has to come first. It's ordered that way, but not exactly clear that that is necessary.

**Comment by cajun-rat** [comment]
Updated

File: docs/linux-secondaries.adoc [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+The `--secondary-network` option enables a second virtual network interface.
+
+Once the secondary is booted, start the primary:
+
+     ../meta-updater/scripts/run-qemu-ota --no-gui --machine qemux86-64 --secondary-network primary-image
+
+Starting aktualizr on the primary will take some time, because of a (suspected) IPv6 bug in systemd.
+
+To check that is is working, run `aktualizr-info` on the primary, and a secondary ECU should be present:
```
Comment (patrickvacek) [inline docs/linux-secondaries.adoc]: "To check that it is working"...

Comment (cajun-rat) [reply docs/linux-secondaries.adoc]: doh

**Comment by cajun-rat** [comment]
3 time lucky

**Pull request merged**